gulpfile.js

```diff
@@ -436,102 +234,26 @@ gulp.task('sync-deps-rollup', (cb) => {
   fs.writeFileSync('./package.json', JSON.stringify(rollupPackage, null, 2), { 'encoding': 'utf8' });
 });
 
-gulp.task('sync-package-service-mapping', (cb) => {
```

Comment:
this task is used to update package_service_mapping.json . This file is used by the docs team to update sdk docs on docs.microsoft.com website. We need to make sure that we update this mapping once we publish packages. I think I missed letting you know about this. It is my fault.

Reply:
Don't worry about it. It's not possible to tell me everything in advance, and even if you tried, I probably wouldn't remember it. This is the perfect time to learn about this. I'll fix it.

Reply:
Done.